docs/merged_objects.md

```diff
@@ -225,8 +225,113 @@ Some merged objects may have some additional sample metadata columns specific to
+In some cases, only some libraries in the merged object will have associated CITE-seq data.
+Libraries which do not have CITE-seq expression data are included in the assay matrices with all `NA` count values.
+
+Only cells which are denoted as "Keep" in the `colData(merged_sce)$adt_scpca_filter` column (as described [above](#singlecellexperiment-cell-metrics)) have normalized expression values in the `logcounts` assay, and all other cells are assigned `NA` values.
+However, as described in the {ref}`ADT data processing section <processing_information:Processed ADT data>`, normalization may fail under certain circumstances.
+If a given ADT library failed normalization, it will also have `NA` values in the merged object.
+
+The following additional per-cell data columns for the ADT data can be found in the main `colData` data frame (accessed with `colData(merged_sce)` [as above](#singlecellexperiment-cell-metrics)).
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `altexps_adt_sum`      | UMI count for CITE-seq ADTs                       |
+| `altexps_adt_detected` | Number of ADTs detected per cell (ADT count > 0 ) |
+| `altexps_adt_percent`  | Percent of `total` UMI count from ADT reads       |
+
+
+In addition, the following QC statistics from [`DropletUtils::cleanTagCounts()`](https://rdrr.io/github/MarioniLab/DropletUtils/man/cleanTagCounts.html), which were calculated on individual libraries before objects were merged, can be found in the `colData` of the `"adt"` alternative experiment, accessed with `colData(altExp(merged_sce, "adt"))`.
+
+| Column name                | Contents                                          |
+| -------------------------- | ------------------------------------------------- |
+| `zero.ambient`   | Indicates whether the cell has zero ambient contamination   |
+| `sum.controls` |  The sum of counts for all control features. Only present if negative/isotype control ADTs were used |
+| `high.controls`  | Indicates whether the cell has unusually high total control counts. Only present if negative/isotype control ADTs were used |
+| `ambient.scale` |  The relative amount of ambient contamination. Only present if negative/isotype control ADTs were not used |
+| `high.ambient`  | Indicates whether the cell has unusually high contamination. Only present if negative/isotype control ADTs were not used |
+| `discard`  | Indicates whether the cell should be discarded based on ADT QC statistics |
+
+
+Metrics for each of the ADTs assayed can be found as a `DataFrame` stored as `rowData` within the alternative experiment:
+
+```r
+rowData(altExp(merged_sce, "adt")) # adt metrics
+```
+
+This data frame contains the following columns with statistics for each ADT.
+The columns `mean-SCPCL000000` and `detected-SCPCL000000` are present for each CITE-seq library in the merged object.
```

Comment:
```suggestion
The columns `mean-SCPCL000000` and `detected-SCPCL000000` are present for each library in the merged object that has associated CITE-seq data.
```

Reply:
Haha, this is hilarious - this was my original sentence and I changed it! Should have stuck with my gut :)